content/docs/support-and-ops/ops-recipes/audit-logs-troubleshooting.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Audit Logs Troubleshooting"
+owner:
+- https://github.com/orgs/giantswarm/teams/team-horizon
+confidentiality: public
+---
+
+How can I check the action of certain user in a cluster?
+
+# Table of Contents
+1. [Gather all logs](#Gather-all-logs)
+1. [Filter out undesired events](#Filter-out-undesired-events)
+
+## Gather all logs
+
+Till we have audit logs in loki installation is handy to have some commands at hand to debug audit logs. First we need to collect all logs available. Since API only retain certain amount of audit logs depending on size, you my not have enough data to get the evidence you are looking too. You can check how old are the log files with `for file in $(kubectl get --raw=/logs/apiserver/ | awk -F'>' '{print $2}' | sed 's/<\/a$//' ); do echo $file; done`. If the event you are looking for is inside the time window then start dumping all data to your local to work more efficient.
```

Comment:
````suggestion
We plan to provide audit log analysis via our Loki installations. Until then, we depend on manual steps to debug them.

To do that, we need to collect all logs available first. Since the backend only retains a certain amount of audit logs (depending on size), your data may not reach far enough into the past to get the evidence you are looking for.

You can check the age of the log files using:
```bash
for file in $(kubectl get --raw=/logs/apiserver/ | awk -F'>' '{print $2}' | sed 's/<\/a$//' ); do echo $file; done
```

If the event you are looking for is inside the time window, dump all data to your local machine for efficient processing.
````